hypervisor/qemu/qemu.go

```diff
@@ -51,10 +51,15 @@ func (qd *QemuDriver) InitContext(homeDir string) hypervisor.DriverContext {
 		os.Mkdir(QemuLogDir, 0755)
 	}
 
-	qemuLogFile := QemuLogDir + "/" + homeDir[strings.Index(homeDir, "vm-"):len(homeDir)-1] + ".log"
-	if _, err := os.Create(qemuLogFile); err != nil {
+	logFile := QemuLogDir + "/" + homeDir[strings.Index(homeDir, "vm-"):len(homeDir)-1] + ".log"
+	if _, err := os.Create(logFile); err != nil {
```

Comment:
sorry, it is not called from vm-association.
qemuLogFile need to be set in the LoadContext()